queries/python/indents.scm

```diff
@@ -91,6 +120,11 @@
   (#set! indent.close_delimiter ")"))
 
 ((parameters) @indent.align
+  (#set! indent.open_delimiter "(")
+  (#set! indent.close_delimiter ")"))
+
+((parameters) @indent.align
+  (#lua-match? @indent.align "[^\n ]%)$")
```

Comment:
At some point, the question becomes: Should this just be done from `indent.lua`, instead of here.

Reply:
I think that would be the best solution, this PR is just the workaround I've been using in my personal config. If you feel strongly about it feel free to close this PR and I'll implement a fix there sometime in the next week or two.